pandas/io/tests/test_excel.py

```diff
@@ -481,6 +481,16 @@ def test_set_column_names_in_parameter(self):
             tm.assert_frame_equal(xlsdf_no_head, refdf)
             tm.assert_frame_equal(xlsdf_with_head, refdf)
 
+    def test_date_conversion_overflow(self):
+        # GH 10001 : pandas.ExcelFile ignore parse_dates=False
+        refdf = pd.DataFrame([[pd.Timestamp('2016-03-12'), 'Marc Johnson'],
+                              [pd.Timestamp('2016-03-16'), 'Jack Black'],
+                              [1e+20, 'Timothy Brown']],
```

Comment:
I am think this would be converted as an integer, right? (or is it actually a float)